src/streaming/Stream.js

```diff
@@ -430,6 +434,26 @@ function Stream(config) {
         }
     }
 
+    function filterCodecs(type) {
+        const realAdaptation = dashManifestModel.getAdaptationForType(manifestModel.getValue(), streamInfo.index, type, streamInfo);
+        let codec;
```

Comment:
`codec` could be moved inside the `.filter` method and declared `const`?

Reply:
Absolutely. Honestly, I thought about that but limited myself to avoid touching too much...

@bbcrddave, thanks for the comment.